app/javascript/styles/bootstrap.scss

```diff
@@ -1,4 +1,5 @@
-// For making bootstrap utilities available.
+@import 'bootstrap/scss/bootstrap';
+// @import "rails_bootstrap_forms";
```

Comment:
Intentionally leaving this commented out right now. Eventually we'll be able to import it more cleanly but for right now it doesn't seem to make that much of a difference, either in terms of display or interaction.